core/src/test/java/org/elasticsearch/action/admin/indices/create/CreateIndexIT.java

```diff
@@ -244,10 +247,15 @@ public void onFailure(Throwable e) {
         for (int i = 0; i < numDocs; i++) {
             try {
                 synchronized (indexVersionLock) {
-                    client().prepareIndex("test", "test").setSource("index_version", indexVersion.get()).get();
+                    client().prepareIndex("test", "test").setSource("index_version", indexVersion.get()).setTimeout(TimeValue.timeValueSeconds(10)).get();
                 }
             } catch (IndexNotFoundException inf) {
-                // fine
+                // fine index has been deleted
+            } catch (ProcessClusterEventTimeoutException ex) {
```

Comment:
In what cases do you we expect a custer event timeout due to index deletion? we do have #14932 , but that's a valid issue?